lib/cfndsl/UpdatePolicy.rb

```diff
@@ -22,8 +22,12 @@ def value
       return @value
     end
     
-    def to_json(*a) 
-      @value.to_json(*a)
+    def as_json(*a) 
+      @value
+    end
+
+    def to_json(*a)
+      as_json.to_json(*a)
```

Comment:
There's a lot of duplication with these two methods:

```ruby
def as_json(*a) 
  @value
end

def to_json(*a)
  as_json.to_json(*a)
end
```

Could we move them into a module? Maybe something like:

```ruby
module CfnDsl
  module JSONSerialisableObject
    def as_json(*a) 
     @value
    end

    def to_json(*a)
      as_json.to_json(*a)
    end
  end 
end
```

then we can just include it where it's duplicated and have the custom behaviour in `lib/cfndsl/JSONable.rb`.